tools/genenotebook/launch_gnb.sh

```diff
@@ -78,4 +80,8 @@ done;
 
 grep -q "Healthcheck OK" ./gnb.log
 
+# More sleep just in case
```

Comment:
Same here, no need for more sleep here probably, and when you have to much sleep the jobs take forever to fail when it happens